lib/travis/api/app/endpoint/jobs.rb

```diff
@@ -140,6 +143,17 @@ def archive_url(path)
       def hostname(name)
         "#{name}#{'-staging' if Travis.env == 'staging'}.#{Travis.config.host.split('.')[-2, 2].join('.')}"
       end
+
+      private def include_log_id?
+        params[:include_log_id] ||
+          !Travis.config.logs_api.enabled? ||
+          request.user_agent.to_s.start_with?('Travis')
+      end
+
+      private def empty_log(job_id)
+        # XXX: Should this empty version be defined elsewhere?
```

Comment:
I tried moving to the service, but then I need to implement an object instead of a plain hash.  I like the idea of Log/RemoteLog knowing how to represent an empty version of itself.  Should that change be included in this PR?